todoapp_server/users/admin.py

```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from .models import User
 
-# Register your models here.
+admin.site.register(User, UserAdmin)
```

Comment:
Ovo nije pod moranje cisto da ne dodje do zabune, ovo je samo da imas preko admin panela mogucnost managa-ovanja user-a

Reply:
Da, citao sam o tome, ali sam hteo malo da probam kako radi taj admin panel pa sam dodao to.